Remove from contrib

MelvinHK opened the pull request:
I appreciate the thought, but I dont feel comfortable having myself on the site's contrib list. Thank you still 🙏

MgramTheDuck commented:
manually merged instead due to conflicts after restructure. Ive pushed the changes into main another way.

MelvinHK committed:
Remove from contrib

I appreciate the thought, but I dont feel comfortable having myself on the site's contrib list. Thank you still 🙏
